.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
There should be a license & copyright here.

Reply:
Done.